### DISCUSSION TITLE
Database backups

### DISCUSSION THREAD

**Discussion by HerrSammyDE** [pelican-dev/panel#536] [Ideas]
It would be great if you could add that you can also create backups of databases

**Comment by fackelmeyer** [comment]
I just came here to request the exact same feature :)
File backups are nice but if someone got access to the database somehow or you messed smth up it would be nice to also keep that backed up

**Comment by notAreYouScared** [comment]
Databases can get quite large and do not conform to the servers disk space allocation. If the user wishes to make a backup of the database, other tools should be used. 

Not saying we can't make an export option for an "on-demand" backup, but at this time nothing is planned.
